spec/db/schema.rb

```diff
@@ -137,15 +137,12 @@
 
   add_foreign_key(:metal, :metal, :column => 'parent_id')
 
-  create_table "metal_hierarchies", :id => false do |t|
+  create_table "metal_hierarchies" do |t|
```

Comment:
You can't delete specific rows without id.  For exemple `hierarchy_class.joins(arel_join).where(where_condition).destroy_all` will not work if the hierarchy table has no id (activerecord will try to do a subquery and match with id and will fail)